tsconfig.json

```diff
@@ -12,10 +12,12 @@
     "rootDir": "src",
     "noImplicitAny": false,
     "declaration": true,
-    "lib": ["es2015.iterable", "es5"]
+    "lib": ["es2015.iterable", "es5"],
+    "outDir": "dist"
   },
   "exclude": [
     "node_modules",
-    "dist"
+    "dist",
+    "**/*.spec.ts"
```

Comment:
I was trying this with the Query library and I was getting a ton of errors from the linter. It must have been because of Webpack.